File: paymentsense-coding-challenge-api/Paymentsense.Coding.Challenge.Api/Models/Country.cs
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Paymentsense.Coding.Challenge.Api.Models
{
    public class Country
    {
        public string Name { get; set; }
        public string Flag { get; set; }
        public string Alpha3Code { get; set; }
        public double Population { get; set; }
        public IEnumerable<string> TimeZones { get; set; }
        public IEnumerable<Currency> Currencies { get; set; }
        public IEnumerable<Language> Languages { get; set; }
        public string Capital { get; set; }
        public IEnumerable<string> Borders { get; set; }
        public IEnumerable<string> CountryBorders { get; set; }
    }

}

File: paymentsense-coding-challenge-website/src/app/models/Country.ts
import { Language } from './Language';
import { Currency } from './Currency';

export class Country {
    name: string;
    flag: string;
    alpha3Code: string;
    population: number;
    timeZones: string[];
    currencies: Currency[];
    languages: Language[];
    capital: string;
    borders: string[];
    countryBorders: string[];
}


